DamageConfiguration damageConfiguration  #   //
HeadAngleRequest headAngleRequest   #         //cabeza
MotionRequest motionRequest  #             //motion
RobotInfo robotInfo  #                     //lo q esta haciendo
uint64 cont

